nxdrive/engine/dao/sqlite.py

```diff
@@ -2175,22 +2185,61 @@ def get_uploads(self) -> Generator[Upload, None, None]:
                 res.uid,
                 Path(res.path),
                 status,
-                engine=res.engine,
+                res.engine,
                 is_direct_edit=res.is_direct_edit,
-                is_direct_transfer=res.is_direct_transfer,
                 progress=res.progress,
                 doc_pair=res.doc_pair,
                 batch=json.loads(res.batch),
                 chunk_size=res.chunk_size,
-                remote_parent_path=res.remote_parent_path,
-                remote_parent_ref=res.remote_parent_ref,
             )
 
+    def get_dt_uploads(self) -> Generator[Upload, None, None]:
+        """Retrieve all Direct Transfer items (only needed details)."""
+        con = self._get_read_connection()
+        c = con.cursor()
+        for res in c.execute("SELECT * FROM Uploads WHERE is_direct_transfer = 1"):
+            yield Upload(
+                res.uid,
+                Path(res.path),
+                TransferStatus(res.status),
+                res.engine,
+                doc_pair=res.doc_pair,
+                batch=json.loads(res.batch),
+            )
+
+    def get_dt_uploads_raw(self) -> Generator[Dict[str, Any], None, None]:
```

Comment:
Check if this is still needed. Now that lot of fixes were done, it may be useless to have another method such this one.